lib/tlitest/test_tlog_rec_session.py

```diff
@@ -22,6 +22,25 @@ def utempter_enabled():
     return 'libutempter.so' in stdout_data
 
 
+@pytest.fixture
+def gen_ssh_key(request):
+    def _del_ssh_key(user):
+        # Remove ssh key to ensure no tests affected later
+        run(f'sed -i "/stdin_test/d" ~{user}/.ssh/authorized_keys', shell=True)
+        run(f'rm -rf ~/.ssh/id_rsa_{user}*', shell=True)
+
+    def _gen_ssh_key(user):
```

Comment:
I think it makes more sense to generate the SSH key as part of `src/tlitest/tlitest-setup` then only for `test_session_record_pipe_io_stdin` pass the SSH command-line options which will use the SSH key. This keeps setup code in one place and allows the tests themselves to be simpler.